jerry-core/jrt/jrt.h

```diff
@@ -40,6 +40,18 @@
 # define __attr_pure___ __attribute__((pure))
 #endif /* !__attr_pure___ */
 
+#ifdef CONFIG_STORE_CONST_IN_ROM
+#ifndef ICACHE_RODATA_ATTR
+#define ICACHE_RODATA_ATTR  __attribute__((section(".irom.text")))
+#endif /* !ICACHE_RODATA_ATTR */
+#ifndef STORE_ATTR
+#define STORE_ATTR __attribute__((aligned(4)))
+#endif /* !STORE_ATTR */
+#define STORE_IN_ROM ICACHE_RODATA_ATTR STORE_ATTR
+#else /* !CONFIG_STORE_CONST_IN_ROM */
+#define STORE_IN_ROM
+#endif /* CONFIG_STORE_CONST_IN_ROM */
```

Comment:
I would prefer a JERRY_CONST_DATA define. Some systems might want to use it for other purposes than moving things into ROM.